modules/vaos/spec/services/v1/fhir_service_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe VAOS::V1::FHIRService do
+  subject { VAOS::V1::FHIRService.new(user) }
+
+  let(:user) { build(:user, :vaos) }
+
+  before { allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token') }
+
+  describe '#make_request' do
+    context 'with an invalid http method' do
+      it 'raises a no method error' do
+        expect { subject.perform(:let, 'Organization/353830') }.to raise_error(
+          NoMethodError
+        )
+      end
```

Comment:
NITPICK: I don't know if you need to test this, since it's hopefully already tested by the inherited parent class. Maybe just test that subject.is_an_instance_of(Common::Client...)?